snapcraft/cli/remote.py

```diff
@@ -124,6 +111,9 @@ def remote_build(
 
     project = get_project()
 
+    if project.info.base not in ("core", "core16", "core18"):
```

Comment:
instead of this check, check for `is None` or we will automatically block dispatching "bare" and "core20" builds when they are ready....

Reply:
Makes sense, updated.